password_security/models/res_users.py

```diff
@@ -81,6 +81,8 @@ def check_password(self, password):
             password_regex.append(r'(?=.*?\W)')
         password_regex.append('.{%d,}$' % company_id.password_length)
         if not re.search(''.join(password_regex), password):
+            self.env.cr.rollback()
```

Comment:
Sorry, but I didn't understand.
Where you have planned add a `savepoint`?

Note: The module `auth_signup` has the `write`:
 -
 https://github.com/odoo/odoo/blob/676ba1bdb/addons/auth_signup/models/res_users.py#L42